numseq/__init__.py

```diff
@@ -1,6 +1,15 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
+"""numseq package
+
+Contains modules that allow to do different mathmatical functions
+
+author: Aaron Jackson
+Github: TimeApollo
+
+"""
+
 __all__ = [
```

Comment:
Nice use of module level dunder names. You can also set \_\_author\_\_ to add your name.